### PR TITLE
(IAC-774) - Use lf encoding for shell files

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,8 +30,6 @@ group :development do
   gem "puppet-module-win-dev-r#{minor_version}", '~> 0.4',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-lint-i18n",                                        require: false
   gem "github_changelog_generator",                              require: false, git: 'https://github.com/skywinder/github-changelog-generator', ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018' if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')
-  gem 'ed25519', '>= 1.2', '< 2.0'
-  gem 'bcrypt_pbkdf', '>= 1.0', '< 2.0'
```
Comment (DavidS) [inline Gemfile:34]: we might need this workaround for a bit longer? y/n?

File: metadata.json [hunk 2]
```diff
@@ -85,6 +85,6 @@
     }
   ],
   "template-url": "https://github.com/puppetlabs/pdk-templates#master",
-  "template-ref": "1.17.0-0-gd3a4319",
+  "template-ref": "heads/master-0-g4151b0a",
```
Comment (DavidS) [inline metadata.json:88]: :+1:

File: .sync.yml [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ---
+.gitattributes:
+  include:
+    '.gitattributes/*': 'eol=lf'
```
Comment (DavidS) [inline .sync.yml:4]: how does this create `files/shell/* eol=lf` above?

Comment (david22swan) [reply .sync.yml:4]: oops

File: Gemfile (hunk 1 above)
Comment (david22swan) [reply Gemfile]: added back

File: .sync.yml (hunk 3 above)
Comment (david22swan) [reply .sync.yml]: fixed

**Pull request merged**